### ISSUE TITLE
Error in combination with middy

### ISSUE THREAD

**Issue by PawelSwiridow** [iopipe/serverless-iopipe-layers#7]
Hello, I'm using middy middleware https://github.com/middyjs/middy to wrap my lambdas.
After serverless-iopipe-layers plugin installation and configuration I'm getting errors:
```
Unable to import module 'iopipe_wrapper': Error
at Function.Module._resolveFilename (module.js:547:15)
at Function.Module._load (module.js:474:25)
at Module.require (module.js:596:17)
at require (internal/module.js:11:18)
```

**Comment by kolanos** [comment]
@PawelSwiridow Thanks for the bug report.

This may be related to #8. Are you setting any package/function `exclude` or `include` directives?

**Comment by PawelSwiridow** [comment]
Hey @kolanos, no, I don't.

**Comment by kolanos** [comment]
@PawelSwiridow We just released v0.2.0. Can you try testing with that and see if that fixes the issue?

**Comment by kolanos** [comment]
@PawelSwiridow Closing this as it should be fixed by v0.2.1. Reopen this issue if you're still having issues with that version.
